mappings/net/minecraft/item/FireworkStarItem.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_1780 net/minecraft/item/FireworkChargeItem
+CLASS net/minecraft/class_1780 net/minecraft/item/FireworkStarItem
```

Comment:
Nice catch. It looks like this was changed in 1.13 as seen in `ItemInstanceTheFlatteningFix`.